main.css

```diff
@@ -1,15 +1,15 @@
 @font-face {
-    font-family: 'kelson light';
-    src: url('https://cdn.rawgit.com/dynobjx/dynobjx.github.io/master/Kelson%20Sans%20Light.otf');
+    font-family: 'signika light';
+    src: url('Signika-Light.ttf');
```

Comment:
Please use the cdn based URL(raw git) of this font file. Use this commit's hash URL https://github.com/dynobjx/dynobjx.github.io/blob/e528b4a80e50161e51d1bef61f0f6a1c7c495d40/Signika-Light.ttf

 Same with lines 7 and 12.